/home/matereza/PycharmProjects/damas-core/venv/bin/python /home/matereza/PycharmProjects/damas-core/py/async_cli/main_test.py
------------------------- test_connections() started -------------------------
Body <html>
	<head>
		<title>damas-core</title>
		<link rel="shortcut icon" href="favicon.ico"/>
		<link href="/style.css" rel="stylesheet" type="text/css"/>
		<script type="text/javascript">//<![CDATA[
			document.addEventListener('DOMContentLoaded', function() {
				document.querySelector('#server').innerHTML = window.location.host;
				//document.querySelector('#srv_js').innerHTML = window.location+'damas.js';
				damas.server = "";
				console.log("Welcome to damas-core API");
			});
		</script>
		<script type="text/javascript" src="/js/damas.js"></script>
	</head>
	<body>
		<h1>damas-core</h1>
		<div class="subtitle">
			<a id="server" href="/">
				damas-core API
			</a>
		</div>
		<nav>
			<a href="/js" title="JavaScript"/><img class="logo" alt="Javascript" src="logo_javascript.svg"/></a>
			<a href="/py" title="Python"/><img class="logo" alt="Python" src="logo_python.svg"/></a>
			<a href="/cli" title="Terminal"/><img class="logo" alt="CLI" src="logo_terminal.svg"/></a>
			<br/>
			<br/>
		</nav>
		<footer>
			<a href="https://github.com/remyla/damas-core/wiki/3-API-reference">ðŸ–¹ API Reference</a>
		</footer>
		<!--
		<a href="https://github.com/remyla/damas-core"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/a6677b08c955af8400f44c6298f40e7d19cc5b2d/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f677261795f3664366436642e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_gray_6d6d6d.png"></a>
		-->
	</body>
</html>

-------------------------------- HTTP  http://localhost:8090 CONNECTION OK --------------------------------
==============================================================================================
==============================================================================================
|| RESPONSE SERVER = <ClientResponse(http://localhost:8090) [200 OK]>
<CIMultiDictProxy('X-Powered-By': 'Express', 'Accept-Ranges': 'bytes', 'Cache-Control': 'public, max-age=0', 'Last-Modified': 'Thu, 20 May 2021 14:07:13 GMT', 'Etag': 'W/"64b-1798a1a0af1"', 'Content-Type': 'text/html; charset=UTF-8', 'Content-Length': '1611', 'Date': 'Sun, 30 May 2021 23:46:41 GMT', 'Connection': 'keep-alive')>
 ||
==============================================================================================
==============================================================================================
there is no call back provided
Traceback (most recent call last):
  File "/home/matereza/PycharmProjects/damas-core/py/async_cli/main_test.py", line 203, in <module>
    loop.run_until_complete(main(loop))
  File "/usr/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/home/matereza/PycharmProjects/damas-core/py/async_cli/main_test.py", line 147, in main
    await test_connections_many(print)
  File "/home/matereza/PycharmProjects/damas-core/py/async_cli/main_test.py", line 63, in test_connections_many
    await asyncio.gather(*tasks)
  File "/home/matereza/PycharmProjects/damas-core/py/async_cli/damas_async.py", line 26, in connect
    async with session.get(self.serverURL, ssl=False) as response:
  File "/home/matereza/PycharmProjects/damas-core/venv/lib/python3.9/site-packages/aiohttp/client.py", line 1117, in __aenter__
    self._resp = await self._coro
  File "/home/matereza/PycharmProjects/damas-core/venv/lib/python3.9/site-packages/aiohttp/client.py", line 544, in _request
    await resp.start(conn)
  File "/home/matereza/PycharmProjects/damas-core/venv/lib/python3.9/site-packages/aiohttp/client_reqrep.py", line 890, in start
    message, payload = await self._protocol.read()  # type: ignore
  File "/home/matereza/PycharmProjects/damas-core/venv/lib/python3.9/site-packages/aiohttp/streams.py", line 604, in read
    await self._waiter
aiohttp.client_exceptions.ServerDisconnectedError: Server disconnected

Process finished with exit code 1
